oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +343,38 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+        String[] statSplit = stat.split(" ");
+        if (status.get("Name") == null || status.get("Name").trim().isEmpty()) {
+            // As per man, the second item is the name
+            if (statSplit.length > 1) {
+                String statName = statSplit[1];
+                // remove leading and trailing parentheses
+                statName = statName.substring(1, statName.length() - 1);
+                status.put("Name", statName);
+            }
+        }
+        if (status.get("State") == null || status.get("State").trim().isEmpty()) {
+            // As per man, the third item is the state
+            if (statSplit.length > 2) {
```

Comment:
While I'm suggesting changing away from split, this could have been included in the above conditional.  But it won't work for names with spaces in them.